sql database/scripts/createDB.sh

```diff
@@ -1,6 +1,6 @@
-#!/bin/sh
+#!/bin/bash
 # run this script to create the barebones database
 
 sqlite3 $1 <<EOF
 create table images (Filename TEXT PRIMARY KEY,GTruth INTEGER,DateCollected TEXT,Prediction INTEGER,Orientation REAL,majorLength REAL,minorLength REAL,Confidence REAL,Height REAL,Width REAL);
```

Comment:
Small niche, but can we do a newline for each column definition for readability?